golang/push-kubernetes.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function download_go_compiler() {
+  wget -q "https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz"
+  tar -C /usr/local -xzf go1.14.2.linux-amd64.tar.gz
+  export PATH=$PATH:/usr/local/go/bin
+}
+
+function clone_and_build_go() {
+  git clone https://go.googlesource.com/go
+  cd go
+  # Tag for kube-cross Docker image.
+  TAG=$(date +v%Y%m%d)-$(git rev-parse --short HEAD)
+  export GOROOT=${PWD}
+  cd src
+  ./make.bash
+  cd ../..
+}
+
+function clone_release() {
+  git clone https://github.com/kubernetes/release.git
+  # Note that you can't really move the tool itself around since it has
+  # references to binaries that live relative to its GOROOT.
+  # This is solved by copying the whole GOROOT directory below.
+  cp -r $GOROOT ./release/images/build/cross
+}
+
+function build_kube_cross() {
+  cd release/images/build/cross
+
+  # Modify Dockerfile to use previously built custom version.
+  # The following assumes that $GOROOT was moved to Dockerfile directory.
+  sed -i 's/FROM golang.*$/FROM buildpack-deps:stretch-scm\
```

Comment:
I think sed supports different separators, `/` is not the best choice here as it requires a lot of escaping.
What about using `#`, i.e. `sed s#...#...#`?

Reply:
If I knew about sed's support for separators other than `/` before rolling out this PR... :grinning: 
Thanks for that. The command is way more readable right now.